[READY FOR REVIEW] Brazilian Portuguese - Accessibility Fundamentals Overview

adielcristo opened the pull request:
[Direct link to preview](https://deploy-preview-948--wai-website.netlify.app/fundamentals/pt-br)

Translate resource pages/fundamentals/index.md to pt-BR.

Resolves https://github.com/w3c/wai-translations/issues/256

israelcefrin commented:
Hello @adielcristo and @remibetin, I’ll be reviewing this translation PR and plan to complete it by December 23rd, 2024.

israelcefrin commented:
Hello @adielcristo, thanks again for the translation. I will add my comments below. Feel free to review them and let me know if you want to discuss any further.

## Introduction to Web Accessibility

1) Introdução à acessibilidade web => Introdução à acessibilidade web (em Inglês)

2) Adjusting “porquê” spelling
In Brazilian Portguese the separated “Por que” has two functions: interrogative or relative, i.e. a question or an explanation related directly to the previous statement . The  sentence is stating a reason or a motive that web accessibility is important. Hence, we need to use the variation: “o porquê”. Otherwise, the meaning of the sentence would be the reason why including “accessibility in context” term is important. \
“Porque/por que/porquê/por quê” it is always tricky, because It is not wrong, it relies on the meaning of the original sentence. The pronnounciation is the same for the 4 variations though. 

Apresenta a acessibilidade web e fornece links para recursos adicionais. Inclui acessibilidade em contexto; *por que a acessibilidade é importante para pessoas*, 
To
Apresenta a acessibilidade web e fornece links para recursos adicionais. Inclui acessibilidade em contexto; *o porquê da acessibilidade ser importante para pessoas*, 


## Acessibilidade: é sobre pessoas
1) Flexing verb - the second sentence in the first paragraph it is a verb (to link) likewise the first sentence which starts with a verb. It is not a noun. Hence, I would recommend:
Explica o aspecto humano da acessibilidade e o papel da acessibilidade na diversidade, igualdade e inclusão (DEI). Links para:
To 
Explica o aspecto humano da acessibilidade e o papel da acessibilidade na diversidade, igualdade e inclusão (DEI). _Vincula_ com OR _Vinculando_ com:

## Princípios de acessibilidade (em Inglês)
1) The noum "aplicações" is not wrong, but it is mostly used in the European Portuguese. The Brazilian Portuguese uses "aplicativos".  Also, "aplicação" has many meanings, therefore it diminishes its clarity.
I would recommend to replace it as follows:
Uma introdução concisa aos requisitos de acessibilidade para sites, _aplicativos_, navegadores 

## Princípios de acessibilidade
### Pesquisa
1) Requisitos da pessoa usuária de acessibilidade - if we state it on that order, it seems that there is a specific "accessibility user" or a user of accessibility.  I would recommend to  adjust it:
A) Requisitos de Acessibilidade para Usuários
Or
B)  Requisitos de Acessibilidade para pessoa usuária

adielcristo commented:
> Hello @adielcristo, thanks again for the translation. I will add my comments below. Feel free to review them and let me know if you want to discuss any further.

Hi @israelcefrin, thanks.

> ## Introduction to Web Accessibility
> 
>     1. Introdução à acessibilidade web => Introdução à acessibilidade web (em Inglês)

Already fixed.

>     2. Adjusting “porquê” spelling
>        In Brazilian Portguese the separated “Por que” has two functions: interrogative or relative, i.e. a question or an explanation related directly to the previous statement . The  sentence is stating a reason or a motive that web accessibility is important. Hence, we need to use the variation: “o porquê”. Otherwise, the meaning of the sentence would be the reason why including “accessibility in context” term is important. 
>        “Porque/por que/porquê/por quê” it is always tricky, because It is not wrong, it relies on the meaning of the original sentence. The pronnounciation is the same for the 4 variations though.
> 
> 
> Apresenta a acessibilidade web e fornece links para recursos adicionais. Inclui acessibilidade em contexto; _por que a acessibilidade é importante para pessoas_,  To Apresenta a acessibilidade web e fornece links para recursos adicionais. Inclui acessibilidade em contexto; _o porquê da acessibilidade ser importante para pessoas_,

Fixed.
 
> ## Acessibilidade: é sobre pessoas
> 
>     1. Flexing verb - the second sentence in the first paragraph it is a verb (to link) likewise the first sentence which starts with a verb. It is not a noun. Hence, I would recommend:
>        Explica o aspecto humano da acessibilidade e o papel da acessibilidade na diversidade, igualdade e inclusão (DEI). Links para:
>        To
>        Explica o aspecto humano da acessibilidade e o papel da acessibilidade na diversidade, igualdade e inclusão (DEI). _Vincula_ com OR _Vinculando_ com:

Fixed.

> ## Princípios de acessibilidade (em Inglês)
> 
>     1. The noum "aplicações" is not wrong, but it is mostly used in the European Portuguese. The Brazilian Portuguese uses "aplicativos".  Also, "aplicação" has many meanings, therefore it diminishes its clarity.
>        I would recommend to replace it as follows:
>        Uma introdução concisa aos requisitos de acessibilidade para sites, _aplicativos_, navegadores

Here in Brazil, the term _aplicativos_ is usually used to refer to mobile apps, while _aplicações_ is used more broadly, including web and desktop. Let's keep it, as the main target is web applications. What do you think

> ## Princípios de acessibilidade
> ### Pesquisa
> 
>     1. Requisitos da pessoa usuária de acessibilidade - if we state it on that order, it seems that there is a specific "accessibility user" or a user of accessibility.  I would recommend to  adjust it:
>        A) Requisitos de Acessibilidade para Usuários
>        Or
>        B)  Requisitos de Acessibilidade para pessoa usuária

Fixed.

Thanks for the review and let me know if there is something else to fix.

remibetin commented:
Thanks very much @adielcristo @israelcefrin for collaborating together in December, and for your patience during this time of limited availability on our side. 

I have added @israelcefrin as translation contributor and will publish the translation very soon.

remibetin commented:
> @remibetin the image at the bottom on the page is missing ALT text in the source. Please, let us know if the image is just decorative or it should have an ALT attribute.

I will double-check and let you know. However, since the original version has an empty `alt` attribute, I will publish the translation as is. We can always update the translation later if needed.

remibetin commented:
> @remibetin the image at the bottom on the page is missing ALT text in the source. Please, let us know if the image is just decorative or it should have an ALT attribute.

@adielcristo I am following up on this comment to confirm the image is decorative and is meant to have an empty alt attribute.
